src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Models/PushTemplateOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using CommandLine;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Tool.Models
+{
+    [Verb("push", HelpText = "Push template image to registry")]
+    public class PushTemplateOptions
+    {
+        [Value(0, Required = true, HelpText = "Image reference: <registry>/<imageName>:<imageTag>")]
+        public string ImageReference { get; set; }
+
+        [Option('i', "InputTemplateFolder", Required = false, Default = ".", HelpText = "Input template folder")]
```

Comment:
If the option "InputTemplateFolder" is not required for pushing, the working folder will be pushed by mistake, which should be an unexpected behavior?

Reply:
For now, if inputFolder not given, files in current folder will be packed.

Reply:
can we make it required?

Reply:
Yes, I also think if it is required, some mistakes by users can be avoided.